src/todoList.viewModel.js

```diff
@@ -3,10 +3,20 @@ import {Todo} from './todo.model.js';
 
 export class ViewModel {
     constructor() {
-        this.todoList = [new Todo("Try javaScript"), new Todo("Buy a Unicorn")]        
+        this.todoList = ko.observableArray([]);        
+        this.inputText = ko.observable("");
     }
 
     toggleChecking(todo) {
         todo.checked(!todo.checked());
     }
+
+    addNewTodo(todo, event) {
+        if(event.keyCode === 13 && this.inputText() !== '') {
+            const text = this.inputText();
+            const todo = new Todo(text);            
+            this.todoList.unshift(todo);                
+            this.inputText("");
```

Comment:
Sometimes you use 'single quotes' and sometimes "double quotes". Please follow one or other approach everywhere to make code style consistent